README-en.md

```diff
@@ -12,11 +12,16 @@ Engineers are humans too. We believe that having a shorter psychological distanc
 
 ## Engineer Positions and Careers
 Engineers have multiple roles such as front-end, back-end, infrastructure, etc. At e-dash, we allow individuals to take on multiple roles according to their aptitude and preferences. We do not deny the desire to develop both front-end and back-end, and there is no problem with choosing to specialize in front-end development. Diversity & inclusion is one of the company's values, and this item applies to the choice of an engineering career.
-
 However, if you want to challenge yourself in a field that you have no experience in, we believe that you need to personally catch up to a certain extent. For example, if someone who has only developed in front-end wants to challenge themselves in Go language development, they need to learn the grammar of Go language and how to use major frameworks on their own.
 
 ## Importance of Documentation
 It is very important to document in order to prevent personalization. Be sure to provide a README in the Git repository, and try to describe the technology used in the relevant repository, the directory structure, and the method of building and operating the application. Be sure to describe the method of constructing the local development environment in the README or the company's internal Wiki, and make sure that new engineers can start implementing immediately.
 
 ## Agile Organization
-We believe that the way the team operates and its rules should be periodically reviewed and improved, including this document. We will discuss and update team management, development styles, and methods of technology selection with the current members at that time. Agile is one of the company's values, and this item is an application of that concept to the management of the engineering organization.
\ No newline at end of file
+We believe that the way the team operates and its rules should be periodically reviewed and improved, including this document. We will discuss and update team management, development styles, and methods of technology selection with the current members at that time. Agile is one of the company's values, and this item is an application of that concept to the management of the engineering organization.
+
+## Continuous Learning and Development
+As a software engineer, where new technologies and information might appear every week or even days, it is important to have a mindset of always learning in order to adapt to the newest trends and best practices. Team members are tasked with dedicating a portion of their working time to focus on learning, as long as it will not have a terrible effect on the development timeline. That newly acquired knowledge can then be shared with the whole engineering team.
```

Comment:
```suggestion
As a software engineer, where new technologies and information might appear every week or even days, it is important to have a mindset of always learning in order to adapt to the newest trends and best practices. Team members are recommended to dedicate a portion of their working time to focus on learning, as long as it will not have a terrible effect on the development timeline. That newly acquired knowledge can then be shared with the whole engineering team.
```